sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LogicalPlan.scala

```diff
@@ -227,7 +227,7 @@ object LogicalPlanIntegrity {
    * this method checks if the same `ExprId` refers to attributes having the same data type
    * in plan output.
    */
-  def hasUniqueExprIdsForOutput(plan: LogicalPlan): Boolean = {
+  def hasUniqueExprIdsForOutput(plan: LogicalPlan): Option[String] = {
```

Comment:
let's update the method comment to mention what the return string value contains, if present? Same for other updated return types.